<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>95</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1">
          <item>
           <widget class="QCheckBox" name="live_checkBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Live</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">streamType_buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="file_checkBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: transparent;
</string>
            </property>
            <property name="text">
             <string>File</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">streamType_buttonGroup</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,1">
          <property name="spacing">
           <number>7</number>
          </property>
          <item>
           <widget class="QPushButton" name="openFile_button">
            <property name="text">
             <string>Open</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Maximum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>90</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="fileName_textEdit">
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1,1">
          <item>
           <widget class="QPushButton" name="start_button">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pause_button">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Pause</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stop_button">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Stop</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="streamType_buttonGroup"/>
 </buttongroups>
</ui>
